source/core/index.ts

```diff
@@ -120,6 +120,7 @@ type CacheableRequestFunction = (
 
 type CheckServerIdentityFunction = (hostname: string, certificate: DetailedPeerCertificate) => Error | void;
 export type ParseJsonFunction = (text: string) => unknown;
+export type StringifyJsonFunction = (object: any) => string;
```

Comment:
```suggestion
export type StringifyJsonFunction = (object: unknown) => string;
```

Makes no practical difference, but we prefer `unknown` whenever possible. Same with the other places `any` is used.